.github/workflows/integration_tests.yaml

```diff
@@ -13,5 +13,17 @@ jobs:
         sudo rm -rf /opt/ghc
     - name: Check out code
       uses: actions/checkout@v2
-    - name: Compile and run tests
-      run: docker build -t mbari_lrauv . -f docker/tests/Dockerfile
+    - name: Compile tests
+      run: docker build -t mbari_lrauv_test . -f docker/tests/Dockerfile
+    - name: Create results directory
+      run: mkdir results
+    - name: Run tests
+      run: docker run -v "$(pwd)/results":/results:rw mbari_lrauv_test
+    - name: Get current dir structure
+      run: find .
+    - name: Upload test results.
+      uses: actions/upload-artifact@v2
+      if: always()
```

Comment:
Is this line needed?

Reply:
Yes. If the above step fails, I still want the logs.